server/account/account.go

```diff
@@ -107,3 +111,59 @@ const (
 	// definition in this list.
 	MaxRule
 )
+
+// ruleNames is a map of rules to names.
+var ruleNames = map[Rule]string{
+	NoRule:            "NoRule",
+	FailureToAct:      "FailureToAct",
+	CancellationRatio: "CancellationRatio",
+	LowFees:           "LowFees",
+	PreimageReveal:    "PreimageReveal",
```

Comment:
This comment primarily applies to the `Rule` enum just above but should also reflect here and below.

The [spec](https://github.com/decred/dcrdex/blob/master/spec/community.mediawiki) orders the rule thus:
1. PreimageReveal
2. FailureToAct
3. CancellationRatio
4. LowFees

Reply:
Also, thoughts on combining these `rule*` maps into 1 `map[Rule]RuleDetail` where `RuleDetail` defines `Name string, Description string, Duration time.Duration`? And then have a single `(r Rule) Detail() RuleDetail` method.

Reply:
The order was wrong!

I rather like the separation currently. We can satisfy the Stringer interface easily. Having three separate maps does seem a little silly though. idk...

Reply:
Can still be similarly easily satisfied:
```
func (r Rule) String() string {
	if detail, ok := ruleDetails[r]; ok {
		return detail.Name
	}
	return "unknown rule"
}
```

Reply:
I combined the maps, but not exporting the types. It does look much better imo.